app/src/main/res/values/strings.xml

```diff
@@ -20,6 +20,8 @@
     <string name="appbar_title_view_notes">View Notes</string>
     <string name="appbar_title_add_notes">Add Notes</string>
     <string name="appbar_title_edit_notes">Edit Notes</string>
+    <string name="menuitem_edit_note">Edit Note</string>
+    <string name="menuitem_delete_note">Delete Note</string>
```

Comment:
I can reuse but I thought one was for Button text (delete_note_button_text) and one is for Menu text (menuitem_delete_note). Would keeping them separate be better in case of a string change in the future?

Reply:
That makes sense. So let's keep these two separate resource.